backend/lib/src/api/routes.rs

```diff
@@ -19,6 +19,10 @@ pub fn routes(services: Services) -> Router {
         )
         .route_layer(DefaultBodyLimit::disable());
 
+    let internal_file_upload = Router::new()
+        .route("/uploads/{file_key}", put(handlers::internal_upload_by_key))
```

Comment:
I don't quite understand this internal path. and also it doesn't seem internal because we don't authenticate it at all.

btw the separate router was because `route_layer` applies to all routes in the router, so if you made a new router just for the body limit you can include it in the previous one